src/Tools/dotnet-monitor/CollectionRules/Options/Actions/CollectDumpOptions.cs

```diff
@@ -36,5 +36,10 @@ internal sealed partial record class CollectDumpOptions : BaseRecordOptions, IEg
         [ValidateEgressProvider]
 #endif
         public string Egress { get; set; }
+
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_CollectArtifactOptions_ArtifactName))]
+        public string ArtifactName { get; set; }
```

Comment:
I'm ok with this level of configuration, but I think long term we'll want a higher tier of configuration on a per artifact type rather than per egress type. We can add that later though.

Reply:
I did add `ArtifactName` to `IEgressProviderProperties`, but I did stop at adding to the `CollectionRuleActionDefaultsOptions`. My initial impression was to avoid allowing users to specify the same default artifact name for all actions, as that probably doesn't make much semantic sense. Let me know if you think otherwise.